ios/ReactNativeHealthkit.swift

```diff
@@ -685,6 +685,7 @@ class ReactNativeHealthkit: RCTEventEmitter {
         store.execute(query);
 
         self._runningQueries.updateValue(query, forKey: queryId);
+        resolve(queryId);
```

Comment:
Looks good! I guess the problem here is if waiting for the promise on the initial setup it will never resolve?

Reply:
fwiw I was running into the same hanging issue (subscribeToChanges promise was never resolving) and this immediately fixed it. It doesn't seem to be breaking anything in my app so far. The callbacks for unsubscribing seem to work as well.